src/calculateRentalCost.js

```diff
@@ -3,8 +3,28 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  } else if (days >= 3) {
+    totalCost -= 20;
```

Comment:
Similarly, avoid using literals in expressions. Define a constant for the value `20` that represents the discount for renting 3 or more days, as per the checklist item 2.